### ISSUE TITLE
Actual exception is hidden when doing must.throw('bla bla')

### ISSUE THREAD

**Issue by koresar** [moll/js-must#4]
My assertion looks like this:
````js
myObject.myFunction.must.throw(/the text I expect/);
````

If myFunction throws a different exception
(like ```TypeError: Cannot read property 'myProperty' of null```)
the must.js hides it from me.

It would be beneficial to see the actual exception stack trace while debugging a test.

**Comment by moll** [comment]
I've actually ended up doing a try-catch myself in tests to have more fine grained assertions on errors.
I'm still not particularly sure how to get test runners to print out nested exceptions like this.

```javascript
var err
try { something() } catch (ex) { err = ex }
err.must.be.an.instanceof(Error)
err.code.must.equal(404)
```

Do you have any suggestions or should we just close this issue until we come up with a better idea?

Thanks!

**Comment by koresar** [comment]
Do whatever you think is reasonable, mate. :) As I'm not using your module. Sorry.

**Comment by moll** [comment]
Fair enough. Do check out the [latest Promise support](https://github.com/moll/js-must/issues/22#issuecomment-111840555) if you're into that sort of thing. ^_^
I'll close this issue for now until further ideas arise. Still, thanks!

**Comment by moll** [comment]
Oh, I actually noticed you did propose a solution in https://github.com/moll/js-must/pull/7/files. Thanks!

**Comment by koresar** [comment]
Ha, I forgot. Sorry. :-)
